### ISSUE TITLE
No locale file for en-gb

### ISSUE THREAD

**Issue by Gromski** [NUKnightLab/TimelineJS3#806]
Date formats for EN are set to US formats. The UK uses date formats like (most of) the rest of the world, i.e. day month year

**Comment by JoeGermuska** [comment]
We would happily take a pull request from someone with local knowledge.

**Comment by Coral-66** [comment]
Hi Gromski
Based in Scotland (almost local), could you provide additional detail on issue.  
Bit confused as both Google Sheets options, Excel Based version I use, and JSON file which 'TimelineJS3' actually uses have specific fields for Year/Month/Day. 
Also tried Google Sheets Demo with date for 30th November 2022 (St. Andrews Day) typed as '30/11/2022' and '11/30/22' on separate events, both displayed as per typed input, i.e. no date conversion

**Comment by Gromski** [comment]
Yep, @JoeGermuska hit the nail on the head - it's the output date formatting. I'll try and create a PR of the correct UK locale file this week.
